core/src/main/java/com/klaytn/caver/crypto/KlayCredentials.java

```diff
@@ -24,20 +24,60 @@
 import org.web3j.crypto.Keys;
 import org.web3j.utils.Numeric;
 
+import java.util.*;
+
 import static com.klaytn.caver.wallet.KlayWalletUtils.*;
 
 public class KlayCredentials {
-
-    private final ECKeyPair ecKeyPair;
+    private final List<ECKeyPair> ecKeyPairForTransactionList;
+    private final List<ECKeyPair> ecKeyPairForUpdateList;
+    private final List<ECKeyPair> ecKeyPairForFeeFeePayerList;
     private final String address;
 
     private KlayCredentials(ECKeyPair ecKeyPair, String address) {
-        this.ecKeyPair = ecKeyPair;
+        this.ecKeyPairForTransactionList = Collections.unmodifiableList(Arrays.asList(ecKeyPair));
+        this.ecKeyPairForUpdateList = null;
```

Comment:
Once we set `null` to variable, we always check `null` if this variable is null or not. How about setting empty list using `collections.emptylist()`?

Reply:
Thanks good point.